public/views/home.tpl

```diff
@@ -13,4 +14,15 @@
   </div>
 
   <div id="content"></div>
+
+  <div id="intro" class="dialog" hidden>
```

Comment:
We have a web component which does all this, but I guess that you want to avoid using that for now.